src/main/java/mate/academy/internetshop/model/Order.java

```diff
@@ -0,0 +1,78 @@
+package mate.academy.internetshop.model;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class Order {
+
+    private static Long id = 0L;
+    private Long orderId;
+    private LocalDate date;
+    private Bucket bucket;
```

Comment:
```suggestion
    private Long userId;
```
замовленню не потрібно знати, з якої корзини він вийшов

Reply:
Done